internal/common/pid_sock_status.go

```diff
@@ -42,19 +52,51 @@ func WriteAndHandleStatusFile() error {
 	if err != nil {
 		return fmt.Errorf("Failed to generate a random number for the unix socket path: '%w'.", err)
 	}
-	statusJson.UnixSocketPath = fmt.Sprintf("/tmp/autograder-%s.sock", unixFileNumber)
+	statusJson.UnixSocketPath = filepath.Join("/tmp", fmt.Sprintf("autograder-%s.sock", unixFileNumber))
 
 	err = util.ToJSONFile(statusJson, statusPath)
 	if err != nil {
-		return fmt.Errorf("Failed to write the pid to the status file: '%w'.", err)
+		return fmt.Errorf("Failed to write to the status file: '%w'.", err)
```

Comment:
Include the path in this error.
(This applies to most errors (and the log) on the status file where you have the path available.)